src/Security/RootPermissions.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace SilverStripe\Security;
+
+/**
+ * For root objects that have permission type "Inherit" determine the result of certain permission checks
```

Comment:
A more correct name is "ParentOfRootPermissions" but that's a bit of a mounthful.